src/fides/api/ops/util/data_category.py

```diff
@@ -1,9 +1,36 @@
 from enum import Enum as EnumType
-from typing import Type
+from typing import List, Set, Type, Union
 
 from fideslang import DEFAULT_TAXONOMY
+from fideslang.models import DataCategory as FideslangDataCategory
+from sqlalchemy.orm import Session
 
+from fides.api.ctl.sql_models import DataCategory as DataCategoryDbModel
 from fides.api.ops import common_exceptions
+from fides.core.config import get_config
+from fides.lib.db.session import get_db_session
+
+
+def get_fides_data_categories_from_db() -> List[FideslangDataCategory]:
+    """
+    Returns each DataCategory from the database as it would have been defined in the
+    default taxonomy.
+    """
+    db: Session = get_db_session(get_config())()
+    return [cat.to_fideslang_obj() for cat in DataCategoryDbModel.all(db)]
+
+
+def get_fides_data_categories_from_taxonomy() -> List[FideslangDataCategory]:
+    """Returns the list of DataCategories from the default Fides taxonomy."""
+    return DEFAULT_TAXONOMY.data_category
+
+
+def get_fides_data_category_superset() -> List[FideslangDataCategory]:
+    """Returns the union of DataCategories from the database and the default Fides taxonomy."""
+    # TODO: De-duplicate this list
```

Comment:
Unfortunately `FideslangDataCategory` doesn't hash straight out the box, which rules out using a `Set` without some tweaks, so I've used a `List` for now.